### ISSUE TITLE
c3c5dbf cause TS crashed

### ISSUE THREAD

**Issue by dnqbob** [OpenRA/OpenRA#20093]
### logs
```
OpenRA engine version {DEV_VERSION}
Tiberian Sun mod version {DEV_VERSION}
Date: 2022-06-27 15:33:33Z
Operating System: Windows (Microsoft Windows NT 10.0.19043.0)
Runtime Version: .NET CLR 6.0.1
Exception of type `System.IO.FileNotFoundException`: File not found: shroud.tem
   at OpenRA.Mods.Common.Graphics.FileNotFoundSequence.OpenRA.Graphics.ISpriteSequence.GetSprite(Int32 frame) in D:\work\Projects\Github\OpenRA\OpenRA.Mods.Common\Graphics\DefaultSpriteSequence.cs:line 119
   at OpenRA.Mods.Common.Traits.ShroudRenderer..ctor(World world, ShroudRendererInfo info) in D:\work\Projects\Github\OpenRA\OpenRA.Mods.Common\Traits\World\ShroudRenderer.cs:line 150
   at OpenRA.Mods.Common.Traits.ShroudRendererInfo.Create(ActorInitializer init) in D:\work\Projects\Github\OpenRA\OpenRA.Mods.Common\Traits\World\ShroudRenderer.cs:line 58
   at OpenRA.Actor..ctor(World world, String name, TypeDictionary initDict) in D:\work\Projects\Github\OpenRA\OpenRA.Game\Actor.cs:line 161
   at OpenRA.World..ctor(ModData modData, Map map, OrderManager orderManager, WorldType type) in D:\work\Projects\Github\OpenRA\OpenRA.Game\World.cs:line 214
   at OpenRA.Game.StartGame(String mapUID, WorldType type) in D:\work\Projects\Github\OpenRA\OpenRA.Game\Game.cs:line 190
   at OpenRA.Game.LoadShellMap() in D:\work\Projects\Github\OpenRA\OpenRA.Game\Game.cs:line 504
   at OpenRA.Mods.Common.LoadScreens.BlankLoadScreen.StartGame(Arguments args) in D:\work\Projects\Github\OpenRA\OpenRA.Mods.Common\LoadScreens\BlankLoadScreen.cs:line 99
   at OpenRA.Game.InitializeMod(String mod, Arguments args) in D:\work\Projects\Github\OpenRA\OpenRA.Game\Game.cs:line 490
   at OpenRA.Game.Initialize(Arguments args) in D:\work\Projects\Github\OpenRA\OpenRA.Game\Game.cs:line 421
   at OpenRA.Game.InitializeAndRun(String[] args) in D:\work\Projects\Github\OpenRA\OpenRA.Game\Game.cs:line 290
   at OpenRA.Launcher.Program.Main(String[] args) in D:\work\Projects\Github\OpenRA\OpenRA.Launcher\Program.cs:line 32
```
### Crash due to c3c5dbfa358fb84b8d7694b002873d9d6bac041f, confirmed by merging this commit and test on openra-bleed

**Comment by abcdefg30** [comment]
This will be fixed by #20087.

**Comment by PunkPun** [comment]
Can this issue be closed now that #20087 got merged?

**Comment by penev92** [comment]
Yes.
